Always show geofences on the Map

Abyss777 opened the pull request:
Geofences are showed by default.

There is strange bug, if I set header in a view config - I get error `header.hide is not a function` If set it manually it is OK. In `State` panel it works OK. Weird.

Button looks alone, but when we move settings there, hope it won't looks so.
Just clear and show again on any geofences updates. Don't think we need spend time on precise synchronizing features and store.

![image](https://cloud.githubusercontent.com/assets/5688080/19267910/dec00b9a-8fca-11e6-9413-be81efdef682.png)

tananaev commented on web/app/view/MapController.js:
```diff
@@ -58,6 +65,20 @@ Ext.define('Traccar.view.MapController', {
+            xtype: 'header',
+            title: Strings.mapTitle,
+            items: [{
+                    xtype: 'button',
```
Indentation is wrong here?

tananaev commented on web/app/view/MapController.js:
```diff
@@ -321,5 +342,55 @@ Ext.define('Traccar.view.MapController', {
+    showGeofences: function () {
+        if (this.lookupReference('showGeofencesButton').pressed) {
+            Ext.getStore('Geofences').each(function (geofence) {
+                var feature = new ol.Feature(Traccar.GeofenceConverter.wktToGeometry(this.getView().getMapView(), geofence.get('area')));
```
This line looks pretty long. Are you sure `jscs` doesn't complain about it?

Abyss777 commented:
`jscs` and `jshint` do not complain about line length but I added new line anyway.
Fixed indentation. Also labels look centered now
![image](https://cloud.githubusercontent.com/assets/5688080/19296593/318ec2f4-9056-11e6-8e1e-6dfc7a84d0a1.png)

tananaev commented:
Perfect, thanks.

Abyss777 commented:
Do not think you have ability to change committer name and mail in this commits , right?
It is lesson to me be more intent next time.

tananaev commented:
No, unfortunately it's not possible to change committer after it's been pushed to remote server. I mean it's technically possible, but it will effectively break history for everybody who's using this project.

Abyss777 commented:
OK. I understand. Thanks.
